hooks/post_gen_project.py

```diff
@@ -73,13 +73,24 @@ def remove_pycharm_files():
         shutil.rmtree(docs_dir_path)
 
 
+def create_devcontainer_bash_history_file():
+    history_dir_path = ".history"
+    if not os.path.exists(history_dir_path):
+        os.mkdir(history_dir_path)
+
+    history_file_path = ".history/bash_history"
+    with open(history_file_path, "a"):
+        pass
```

Comment:
I may be missing something, but we usually add the files to the template, and *remove* them in post-gen hooks when they don't apply. Why is this not following the pre-existing pattern? It's *creating* it when needed